### PR TITLE
V3 SysEng Check Edits

### PR DESCRIPTION
**Pull request by chuehlien**
This PR address comments from the Spec Review team that are specific to WZDx v3.0.

- Updating ERD diagram to include legend for boldface text.
- Clarified relationships between road_event_data_sources and road_events.
- Clarified relationships between road_events and lanes.
- Clarified relationships between road_event_data_sources and road_event_feed_info in GeoJSON.

Edit:
- Also updated datetime format to follow RFC 3339 section 5.6 specifically, instead of ISO 8601

### CODE REVIEW

File: spec-content/data-tables/road_event_feed_info.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 The  `road_event_feed_info` table stores feed header information such as metadata, contact information, and data sources.
 
 - A WZDx feed must contain a single `road_event_feed_info` entry.
-- A WZDx feed must contain at least one `road_event_data_source` entry, linked to the `road_event_feed_info` via the `feed_info_id` foreign key on the `road_event_data_sources` table.
+- A WZDx feed must contain at least one `road_event_data_source` entry, linked to the `road_event_feed_info` via the `feed_info_id` foreign key on the `road_event_data_sources` table. In the GeoJSON feed output, each `road_event_data_source` entry is presented as an object in the `data_sources` array in the `road_event_feed_info` properties.
```
Comment (j-d-b) [inline spec-content/data-tables/road_event_feed_info.md:7]: Good addition!

**Review by j-d-b — COMMENTED** [review]
Is it easy for you to move the "Note: required data elements are in bold" in the ERD to the bottom (e.g. bottom left) rather than at the start? Very minor and NBD if it's a pain.

**Review by Mahsa-Ettefagh — APPROVED** [review]

**Review by DeraldDudley — APPROVED** [review]
Looks good.  Thanks.

**Review by j-d-b — APPROVED** [review]
Great @chuehlien! Note I did a very minor rewording for the RFC reference in the schema.

**Pull request merged**